### PR TITLE
started the quiz

### PR DESCRIPTION
**Pull request by jmphilips**

### CODE REVIEW

**Comment by jmphilips** [comment]
@morecallan

File: limit-ten.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+let count = 0;
+
+transformStream._transform = (chunk, _, callback) => {
+
+	processedChunk = chunk.toString() + "\n"
+
+	if (processedChunk == "No Match Found\n") {
+		processedChunk = null
+	}
+ddddd
```
Comment (morecallan) [inline limit-ten.js]: does this not break stuff?

**Comment by morecallan** [comment]
I'm liking this so far! Solid es6 usage, clear variable names; almost at MVP.

Before you get this merged, take a look at a couple things:
- Is the user getting a helpful usage message (see Scott's note) when they enter no args?
- You are handling one scenario for case sensitivity, but what happens if I type "HELLO" in the search?

Looks like you've got a good handle on this, shoot for the bonuses once you wrap this up.

**Comment by jmphilips** [comment]
@morecallan I added a usage statement.  And I cleaned up things a little bit.

**Comment by morecallan** [comment]
GORGEOUS. Great response to comments. Comment it, merge it. 👍

**Pull request merged**